apps/rush-lib/src/logic/base/BaseInstallManager.ts

```diff
@@ -215,8 +251,10 @@ export abstract class BaseInstallManager {
         }
       }
 
-      // Create the marker file to indicate a successful install
-      this._commonTempInstallFlag.create();
+      // Create the marker file to indicate a successful install if it's not a filtered install
```

Comment:
Yes, it does. This is intentional, since each filtered install is technically an "incomplete" install. If you're not running with `--frozen-lockfile` for example, the lockfile in `common/temp` is rewritten to match the install that was last performed (otherwise it remains the same). Given this approach, it felt best to assume that we can only trust that the last install performed was the correct one in the case of filtered installs.

If a dev wants to have multiple projects installed using the filter, that would be done by providing multiple `--to` flags